src/js/edu-benefits/5490/config/form.js

```diff
@@ -67,7 +85,140 @@ const formConfig = {
+                }
+              }),
+              'ui:options': {
+                hideIf: (fieldData) => fieldData.relationship !== 'spouse'
+              }
+            },
+            relativeFullName: _.assign(fullName.uiSchema, {
+              'ui:title': 'Name of Sponsor'
+            }),
+            // This isn't running validation for some reason...
+            veteranSocialSecurityNumber: ssn.uiSchema,
+            veteranDateOfBirth: date.uiSchema('Date of Birth'),
+            veteranDateOfDeath: date.uiSchema('Date listed as MIA, POW, or as deceased'),
+          },
+          schema: {
+            type: 'object',
+            definitions: {
+              'ui:title': 'Sponsor Service',
+              date: date.schema // For spouseInfo
+            },
+            properties: {
+              relationship: _.assign(relationship, {
+                enumNames: enumToNames(relationship.enum, relationshipLabels)
+              }),
+              spouseInfo,
+              relativeFullName: fullName.schema,
+              veteranSocialSecurityNumber: ssn.schema,
+              veteranDateOfBirth,
+              veteranDateOfDeath
+            }
+          }
+        },
+        sponsorService: {
+          title: 'Sponsor Service',
+          path: 'military-service/sponsor-service',
+          uiSchema: {
+            serviceBranch: {
+              'ui:title': 'Branch of service'
+            },
+            currentlyActiveDuty: {
+              'ui:title': 'Is the qualifying individual on active duty?',
+              'ui:widget': 'yesNo'
+            },
+            outstandingFelony: {
+              'ui:title': 'Do you or the qualifying individual on whose account you are claiming benefits have an outstanding felony and/or warrant?',
+              'ui:widget': 'yesNo'
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              serviceBranch,
+              currentlyActiveDuty,
+              outstandingFelony
+            }
+          }
+        },
+        applicantService: {
+          title: 'Applicant Service',
+          path: 'military-service/applicant-service',
+          initialData: {
+            // I'd like to default the checkbox to true...
+            // applyPeriodToSelected: true
+          },
+          uiSchema: {
+            toursOfDuty: (() => {
+              // Set and show the title
+              const uiSchema = toursOfDuty.uiSchema;
+              uiSchema['ui:title'] = 'Applicant service periods';
+              uiSchema['ui:options'].hideTitle = false;
+
+              // Set other labels
+              uiSchema.items = _.assign(uiSchema.items, {
+                serviceStatus: {
+                  'ui:title': 'Service Status'
+                }
+              });
+              return uiSchema;
+            })()
+          },
+          schema: {
```

Comment:
Ahem...yes, yes I did...They were in there until I changed the way I was making the schema (with the optional fields and all), then I forgot to put them back in...0.o